src/server/views/lists/partials/translators-interpreters/translators-interpreters-results-list.njk

```diff
@@ -12,16 +12,36 @@
   <p class="govuk-body">We couldn’t find any results that match your search terms. Try changing these to search again.</p>
 {% else %}
   <h2 class="govuk-heading-l govuk-!-margin-bottom-4">{{resultsTitle}}</h2>
-  <p class="govuk-body">You should carry out your own research before choosing a service provider.</p>
 
-  <div class="govuk-warning-text">
-    <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
-    <strong class="govuk-warning-text__text">
-      <span class="govuk-warning-text__assistive">Warning</span>
-      Translators and interpreters must be authorised to operate in some countries (known as sworn translators).
-      Check providers are qualified to carry out the service you need in the country you have selected.
-    </strong>
-  </div>
+  {% if hasSworn.translatorsAndInterpreters or hasSworn.interpreters or hasSworn.translators %}
+    <div class="govuk-warning-text">
+      <span class="govuk-warning-text__icon" aria-hidden="true">!</span>
+      <strong class="govuk-warning-text__text">
+        <span class="govuk-warning-text__assistive">Warning</span>
+        {% if hasSworn.translatorsAndInterpreters %}
+          In {{ country }}, translators and interpreters must be authorised to carry out legal or official services.
+          <p>These are known as ‘sworn’, ‘certified’ or ‘court’ translators or interpreters and are listed in the results as providing official services.</p>
+        {% elif hasSworn.interpreters %}
+          In {{ country }}, interpreters must be authorised to carry out legal or official services.
+          <p>These are known as ‘sworn’, ‘certified’ or ‘court’ interpreters and are listed in the results as providing official services.</p>
+        {% elif hasSworn.translators %}
+          In {{ country }}, translators must be authorised to carry out legal or official services.
+          <p>These translators are known as ‘sworn’ or ‘certified’ translators and are listed in the results as providing official services.</p>
```

Comment:
These seem to all be the same string, apart from the translators/interpreters/translators and interpreters part. 

How about just assigning a variable? 
```suggestion
          {% set swornServices %}
            {% if hasSworn.translatorsAndInterpreters %}
              translators and interpreters
            {% elif hasSworn.interpreters %}
              interpreters
            {{# etc }}
          {% endset %} 
          In {{ country }}, {{swornServices}} must be authorised to carry out legal or official services.
          <p>These are known as ‘sworn’, ‘certified’ or ‘court’ translators or interpreters and are listed in the results as providing official services.</p>
        
```

Reply:
Yep this is actually much cleaner. My .njk game is weak 😓

Reply:
btw it didn't all start off as the same string, there were a lot of changes to the copy but it should have been refactored.

Reply:
All good. It’s something I picked up from php/Wordpress days so probably for the best..

If hasSworn was an array eg [translators, interpreters] you can just do a .join(“and”) for the set instead of the elifs.

Reply:
Actually, I've noticed for both (interpreters & translators) and interpreters only, the copy contains the word `court` but for translators it doesn't. Would it make sense to have two variables?

Reply:
sure. you can do something like {% set alsoKnownAs = ".." %}. Change to something like

```
{% if hasSworn.translatorsAndInterpreters %}
  {% set alsoKnownAs = ".." %}
  {% set servicesProvided = ".." %}
{% elif %}

```